docs/FAQ.md

```diff
@@ -17,6 +17,33 @@ with `collect`, `show` or `write` a new `DataFrame` is constructed causing Spark
 query. This is why `spark.rapids.sql.enabled` is still respected when running, even if explain shows
 stale results.
 
+### Why does the plan for the GPU query look different from the CPU query?
+
+Typically, there is a one to one mapping between CPU stages in a plan and GPU stages.  There are a
+few places where this is not the case.
+
+* `WholeStageCodeGen` - The GPU plan typically does not do code generation, and does not support generating code for an
+entire stage in the plan. Code generation is typically used to reduce the cost of processing data
+one row at a time. The GPU plan processes the data in a columnar format, so the costs are different.
+
+* ColumnarToRow and RowToColumnar Transitions - The CPU version of Spark plans typically process data in a row based format. The main exception to
```

Comment:
Should ColumnarToRow and RowToColumnar be backquoted, like, for example, `WholeStageCodeGen` is in the previous section?  

```suggestion
* `ColumnarToRow` and `RowToColumnar` transitions - The CPU version of Spark plans typically process data in a row based format. The main exception to
```